bin/owl_list_gist.sh

```diff
@@ -12,9 +12,15 @@ fi
 DIRS=`ls -d $DIRNAME/* | xargs echo`
 
 for DIR in $DIRS; do
+  if [ -f $DIR ]; then
+    continue
+  fi
   ID=`basename $DIR`
-  if [ -f $DIR/readme.md ]; then
-    INFO=`head -n 1 $DIR/readme.md`
+  SUBDIR=`ls -dt $DIR/* | head -n 1 | xargs echo`
+  if [ -f $DIR/\#readme.md ]; then
+    INFO=`head -n 1 $DIR/\#readme.md`
+  elif [ -f $SUBDIR/\#readme.md ]; then
+    INFO=`head -n 1 $SUBDIR/\#readme.md`
```

Comment:
Oh, I just noticed, why the head file is still here? Didn't we agree on moving this into ocaml code? Can you fix this!